projects/1.16/assets/coffee-mod/coffeemod/lang/zh_cn.json

```
@@ -1 +1,6 @@
-{}
\ No newline at end of file
+{
+  "block.coffeemod.coffee_bush": "咖啡种子",
+  "block.coffeemod.coffee_machine": "咖啡机",
+  "item.coffeemod.roasted_coffee_beans": "烤咖啡豆",
```

```suggestion
  "item.coffeemod.roasted_coffee_beans": "熟咖啡豆",
```
也可以叫咖啡熟豆

---

熟咖啡豆吧

---

或者烘焙咖啡豆

---

熟咖啡豆可能会有歧义

---

就是这个果子是熟的还是他是烤（烘焙）过的

---

把这个suggestion改一改我来commit

---

```suggestion
  "item.coffeemod.roasted_coffee_beans": "烘焙咖啡豆",
```
https://en.wikipedia.org/wiki/Coffee_bean#Composition
> The term "green coffee bean" refers to unroasted mature or immature coffee beans. These have been processed by wet or dry methods to remove the outer pulp and [mucilage](https://en.wikipedia.org/wiki/Mucilage) and have an intact wax layer on the outer surface.